src/components/EditableField.jsx

```diff
@@ -134,7 +134,7 @@ const EditableField = (props) => {
             deleteEnabled
               ? (
                 <Tooltip placement='top' title='Delete' mouseLeaveDelay={0}>
-                  <Button size='small' shape='circle' icon={<DeleteOutlined />} onClick={deleteEditableField} />
+                  <Button aria-label='Delete' size='small' shape='circle' icon={<DeleteOutlined />} onClick={deleteEditableField} />
```

Comment:
what are these labels for? Are they for the integration testing?

Reply:
they are for accessibility, so screenreaders can find a label for the button (since the text is just an SVG). they also help locate the item for unit testing, yes